srv/components/system/config/set_hostname.sls

```diff
@@ -15,9 +15,12 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+Update hostname file permissions:
```

Comment:
could you make that more specific, e.g. `Make hostname file modifiable`

Reply:
Updated